t/decode-transactions

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# For every create-withdrawal-data.*.golden, use bitcoin-cli
+# "decoderawtransaction" to dump the contents of the constructed
+# transaction.
+#
+# I'm using this to document the changes that Bitcoin Core v0.17 is
+# making to the transactions (shorter signatures thanks to
+# https://github.com/bitcoin/bitcoin/pull/13666)
+#
+
+bitcoind -testnet -daemon -connect=0.0.0.0
+# Give bitcoind time to start up
+sleep 5
+
+
+
+for i in create-withdrawal-data.*.golden; do
+    # Find longest line (which will be the transaction hex)
```

Comment:
we should make this more robust and parse the structure for the correct key. I'm ok with this being a followup issue/PR if you'd prefer.

Reply:
I will look into this next week...